Milestone 1: Project Setup

AsumVictor opened the pull request:
# Milestone 1: Project Setup

 # Milestone 3: List interactive
 In this PR, I have added some files and configurations to set up the leadership board project.
 
 ## Type of change
 
 * [ ] Install and configure webpack
 * [ ] Install and configure tailwindcss
 * [ ]  Created a directory to store all modules
 * [ ] Add a function to display the content of the table dynamically with score data.
 
 # Checklist:
 * [ ]  My code follows the style guidelines of this project
 * [ ] I have performed a self-review of my code
 * [ ] There is no linters error in my code
 * [ ]  My changes generate no new warnings

AsumVictor committed:
Setup webpack

AsumVictor committed:
webpack and files setup

AsumVictor committed:
Add tailwind css, Created module folder

AsumVictor committed:
Display table dynamically on the screen

AsumVictor committed:
Update README file

divinecharlotte reviewed (changes requested):
Hi @AsumVictor  , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- Descriptive pr 👍 
- Descriptive readme 👍 

 




 
![](https://media.giphy.com/media/tKyrc0NTlEcGWdThp4/giphy.gif)
## Required Changes ♻️

- [x] kindly consider adding a working link to your LinkedIn account under the author's section to make your readme professional and interactive
- [x] kindly consider using the correct git-flow as requested in the project requirement to protect your main branch 
- hint : check this[ link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md) as a reminder

- [x] kindly consider giving your project  the below basic structure 
```
app-root-directory
  |- package.json
  |- webpack.config.js
  |- /dist
    |- bundle.js or be main.js
    |- index.html
  |- /src
   |- style.css
    |- index.js
  |- /node_modules
```
to make sure that all the required files using Webpack are available  and webpack is used to bundle JavaScript.  (If you need a refresher, check the previous project or Webpack exercise).

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AsumVictor committed:
Update README.md

AsumVictor committed:
Add dist directory to the root for the basic file structure

AsumVictor commented:
Thank you very much for the review!
##Changes
* [ ] I have updated the readme file with the working LinkedIn link
* [ ] I have added the dist repository for the basic structure
* [ ] I have created the development branch

muneebulrehman reviewed (approved):
Hi @AsumVictor , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Descriptive PR
- Good follow-up with the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
